.github/workflows/release.yml

```diff
@@ -1,4 +1,21 @@
-name: "Publish: manual full release OR automatic snapshot"
+# Release artifact will be pushed to Sonatype, which is synced to Maven Central
+# Build artifacts get pushed to Sonatype and non-SNAPSHOT versions are then
+# auto-synced to Maven Central.
+#
+# If images are given (separated by spaces), then they are assumed to be produced by
+# Maven and have release and dev tags.
+#
+# Various secrets are needed to do the release.
+#
+# Credentials for Sonatype
+#   - OSSRH_USERNAME
+#   - OSSRH_PASSWORD
+# GPG key + passphrase to sign artifacts for Maven Central
+#   - MAVEN_GPG_PASSPHRASE
+#   - MAVEN_GPG_KEY
+
+name: "Publish: manual full release"
```

Comment:
Should we remove possible ambiguity between this and the `pass-java-release` names, maybe naming this one "Release `eclipse-pass/main`"?

Reply:
If this gets changed then all of the release.yml in the other repos should be changed as well. I am agnostic.

Reply:
Gotcha, I hadn't seen the workflows in the other repos yet. In that case, we might as well keep this name generalized